README.md

```diff
@@ -33,6 +33,21 @@ On NixOS and nix-darwin, simply replace your `nix.package`, like so:
 ```nix
 nix.package = pkgs.nix-monitored;
 ```
+And then import and enable it like this:
+```nix
+{
+  imports = [
+    inputs.nix-monitored.nixosModules.default
+  ];
+
+  nix.monitored.enable = true;
+}
+```
```

Comment:
Importing and enabling the module actually replaces setting `nix.package` manually, as that's part of the module's implementation: https://github.com/ners/nix-monitored/blob/master/flake.nix#L108

I'd also like to mention that there is a Darwin module, not just a NixOS one.